### DISCUSSION TITLE
中央大學學生沒機車是不是跟殘廢沒兩樣

### DISCUSSION THREAD

**Discussion by Jinyuu17** [1101-GS4719-C/Python-Community-GS4719-C#2231] [General]
眾所周知，後門和宵夜街的物家堪比台北市，被中原很甩十條街。想下山去吃飯又要看桃園公車的臉色，脫班嚴重，這對於沒機車的人來說相當痛苦。所以你們覺得在中央需要有機車嗎。

**Comment by yuan3x2** [comment]
我自己是認為蠻需要的，有機車可以方便接家教找工作賺外快，還可以節省交通時間，但缺點就是要多花一筆錢，或許可以考慮畢業二手賣，整體來說算是利大於弊？

**Comment by Tchang323** [comment]
我也覺得蠻需要的，但家人不准騎機車，只能靠朋友XD。我感覺如果男生有機車會蠻加分的，可以帶女朋友去約會，你要不要考慮一下咧?

**Comment by gigitilto** [comment]
我認為有些人需要，而另一群人不需要。對於那些需要的人來說，他們一定是有用途，以我的室友來說，他常常騎車去兜風，去聚餐也都非常方便，但對我這種不需要的人來說，除了不常下山之外，火車站說實話離我們學校很近，騎ubike很快就到了，所以只要活動範圍只有在學校和中央之間又沒錢也沒空保養的同學，是不必買車的。

**Comment by Boying12** [comment]
首先，機車能大大提高出行的靈活性，無論是想去後門、宵夜街，還是其他地方吃飯，擁有機車可以隨時出發，不用再依賴公車的班次和時間。這樣不僅節省了等待的時間，也能讓你隨心所欲地探索周邊的美食，享受更多的生活樂趣。
其次，對於參加各種活動和社團，機車也能讓你更方便地前往不同的地點，增強參與感。無論是校內的活動還是校外的聚會，機車都能幫助你更輕鬆地到達，讓你不錯過任何精彩的瞬間。
最後，騎機車的過程也是一種放鬆的方式，尤其是在美麗的校園和周邊路線上，騎行的時光能讓你暫時忘卻學業的壓力，享受自由的感覺。

**Comment by yue083139** [comment]
我個人是覺得在中央有機車蠻方便的，可以比較好接家教，有時候突然想吃點好料的也能夠說走就走不會花你太多時間，畢竟中央的公車是真的不太方便，路線不多班次少尖峰時段還一堆人擠，有機車想甚麼時候去甚麼時候回來都行，但是價格的部分可能要稍微考慮就是了。

**Comment by qiu0109** [comment]
有機車可以隨時下山吃美食、跟朋友逛街，如果有接家教的話也很方便，但我也有認識沒有機車卻常常往外跑的人，所以我覺得對部分人來說機車並不是必要的，還是有其他方式可以到達想去的地方。而且如果不是經常需要下山的人，騎goshare也是代替買機車的好方法。

**Comment by kzx-creat** [comment]
反正學校後門有goshare，大大拯救想利用機車通勤或出去覓食的同學。但我自己是沒使用過，所以沒辦法比較騎自己的機車跟goshare的差別，不知道有沒有不好租到車之類的問題？

**Comment by Mminniehuang** [comment]
我覺得是還沒到沒有機車會活不下去的程度，但是如果有真的非常方便非常棒。除了飲食自由之外，我覺得可以隨時逃離喘不過氣的學習環境也是一點，不管是出門買買東西、吃個飯、看個夜景，或是漫無目的的晃晃都是我覺得很好的充電方式。

**Comment by Elaine4488** [comment]
在中央大學擁有自己的機車確實會有一些方便之處，但也有一些需要考慮的優缺點。我覺得擁有機車確實可以提高生活便利性，特別是在時間上，但也要權衡停車、維護和安全等方面的問題。如果主要是校內活動和上課，若公共交通或步行能解決問題，機車的需求可能就沒那麼大。

**Comment by poyen0806** [comment]
我不太懂為什麼要這樣說，後門有go share可以租，所以已經可以應付大多數地方了，雖然有些地方不支援go share，但說成殘廢也太超過了。而且公車也十分方便，機車的需求沒那麼大。如果不是經常需要下山的人，基本上不需要機車。
